Empire - The Last Battle/Assets/Unity/Scripts/MonoBehaviours/BattleUnitPositionManager.cs

```diff
@@ -1,110 +1,126 @@
 ﻿using UnityEngine;
 using System.Collections.Generic;
 
-public class BattleUnitPositionManager : MonoBehaviour {
+public class BattleUnitPositionManager : MonoBehaviour
```

Comment:
There shouldn't be all these changes. it's hard to see what the actual changes were. You should fix your editor so that it doesn't do autoformatting.